moveit_planners/chomp/chomp_interface/src/chomp_interface.cpp

```diff
@@ -71,8 +70,6 @@ void CHOMPInterfaceROS::loadParams() {
   nh_.param("collision_threshold", params_.collision_threshold_, 0.07);
   nh_.param("random_jump_amount", params_.random_jump_amount_, 1.0);
   nh_.param("use_stochastic_descent", params_.use_stochastic_descent_, true);
-  //filter_mode_ = false;
```

Comment:
These parameters should be documented and perhaps namespaced under a "chomp" namespace.